.github/workflows/ui-tests.yaml

```diff
@@ -34,10 +36,33 @@ jobs:
         run: |
           sudo echo "127.0.0.1 local.kyma.dev" | sudo tee -a /etc/hosts
           wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | sudo bash
-          IMG=europe-docker.pkg.dev/kyma-project/dev/api-gateway-manager:PR-${{ github.event.number }} ./tests/ui/tests/scripts/k3d-ci-kyma-dashboard-integration.sh
+          IMG=europe-docker.pkg.dev/kyma-project/dev/api-gateway-manager:PR-${{ github.event.number }} ./tests/ui/tests/scripts/k3d-ci-kyma-dashboard-integration.sh prod
       - uses: actions/upload-artifact@v4
         if: always()
         name: Export Cypress output
         with:
           name: cypress
           path: tests/ui/tests/cypress/
+
+  run-ui-tests-periodic:
+    name: Run UI tests
+    if: github.event_name == 'schedule'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+      - name: Run tests
+        run: |
+          sudo echo "127.0.0.1 local.kyma.dev" | sudo tee -a /etc/hosts
+          wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | sudo bash
+          IMG=europe-docker.pkg.dev/kyma-project/dev/api-gateway-manager:PR-${{ github.event.number }} ./tests/ui/tests/scripts/k3d-ci-kyma-dashboard-integration.sh stage
```

Comment:
I think we must not use the tag `PR-${{ github.event.number }}` for api-gateway-manager, but use the latest version from main.

Reply:
I've just separated two workflows, since in 2nd look it looks cleaner this way. Also fixed those oversights.

Reply:
Also seeing skipped job all the time on PR wouldn't be too nice.